halo2_proofs/src/dev.rs

```diff
@@ -670,6 +672,37 @@ impl<F: Field + Group> Assignment<F> for MockProver<F> {
     }
 }
 
+fn home_made_par_iter_flat_map<U: Send, Fun: Send + Sync, const ISPAR: bool>(
```

Comment:
I suggest changing the name to something like: `parallelizable_iter_flat_map` or `custom_par_iter_flat_map`